src/main/java/com/denizenscript/depenizen/bukkit/properties/residence/ResidenceLocationProperties.java

```diff
@@ -4,11 +4,11 @@
 import com.bekvon.bukkit.residence.protection.ClaimedResidence;
 import com.denizenscript.denizencore.objects.properties.Property;
 import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
 import com.denizenscript.depenizen.bukkit.objects.residence.ResidenceTag;
 import com.denizenscript.denizen.objects.LocationTag;
 import com.denizenscript.denizencore.objects.core.ElementTag;
 import com.denizenscript.denizencore.objects.ObjectTag;
-import com.denizenscript.denizencore.tags.Attribute;
 
 public class ResidenceLocationProperties implements Property {
```

Comment:
This (and `ResidencePlayerProperties`) are extension properties, properties used for the sole purpose of adding tags / mechs; the modern alternative to that is extension classes - [here's](https://github.com/DenizenScript/Denizen/blob/d2469a1e9fd745721e1ff2b126438264c760bbeb/plugin/src/main/java/com/denizenscript/denizen/objects/properties/bukkit/BukkitScriptExtensions.java) an example, but feel free to ask on Discord if you need any help converting them